app/controllers/labels_controller.rb

```diff
@@ -3,6 +3,7 @@ class LabelsController < SitesController
   def edit
     add_breadcrumb t(:'hyrax.controls.home'), root_path
     add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
+    add_breadcrumb t(:'hyrax.admin.sidebar.configuration'), '#'
```

Comment:
Unrelated to this PR, but this is a menu item adjacent to content blocks and pages, so I thought I'd add this breadcrumb for consistency with sibling menu items.